WindowsServerDocs/administration/performance-tuning/additional-resources.md

```diff
@@ -115,3 +115,5 @@ Here's a list of performance tuning guidelines for previous versions of Windows
 -   [Performance Tuning Guidelines for Windows Server 2008 R2](https://download.microsoft.com/download/6/B/2/6B2EBD3A-302E-4553-AC00-9885BBF31E21/Perf-tun-srv-R2.docx)
 
 -   [Performance Tuning Guidelines for Windows Server 2008](https://download.microsoft.com/download/9/c/5/9c5b2167-8017-4bae-9fde-d599bac8184a/Perf-tun-srv.docx)
+
+-   [Performance Tuning Guidelines for Windows Server 2016](https://docs.microsoft.com/en-us/windows-server/opbuildpdf/administration/performance-tuning/toc.pdf?branch=live)
```

Comment:
The link works as intended. Links to docs.microsoft.com can (in almost all cases) are usually better off without the locale part (/en-us/):
```suggestion
-   [Performance Tuning Guidelines for Windows Server 2016](https://docs.microsoft.com/windows-server/opbuildpdf/administration/performance-tuning/toc.pdf?branch=live)
```
I have tested that it works without /en-us/, but feel free to skip this change, if you find that the link can break randomly without the locale part.

Reply:
Thanks, updated!